pkg/cmd/install.go

```diff
@@ -129,7 +131,19 @@ func loadPublickey(path string) ssh.AuthMethod {
 	}
 	signer, err := ssh.ParsePrivateKey(key)
 	if err != nil {
-		panic(err)
+		if err.Error() != "ssh: cannot decode encrypted private keys" {
+			panic(err)
+		}
+
+		fmt.Printf("Enter passphrase for '%s': ", path)
+		bytePassword, err := terminal.ReadPassword(int(syscall.Stdin))
```

Comment:
Thanks for submitting a PR.

I think I'd rather take this in via `stdin` so that it can be automated and run headless.

I.e.

https://github.com/openfaas/faas-cli/blob/0e810b520aee78ff4ede68e67068a652df395810/commands/login.go#L68

Restic for instance can read via password file, stdin and environment file:

https://restic.readthedocs.io/en/latest/faq.html#how-can-i-specify-encryption-passwords-automatically

Reply:
Im not sure if thats best practice for ssh passphrases.

Maybe we should ask the ssh-agent for the passphrase if it is available and let the user enter the password otherwise as a fallback.

I also think that would be the behavior that the user expects from the experience with ssh clients.

Reply:
```
./k3sup install --ip $IP --user root
Public IP: ......
ssh -i /home/rkaufmann/.ssh/id_rsa root@......
Enter passphrase for '/home/rkaufmann/.ssh/id_rsa': ddd
Error: unable to load the ssh key with path "/home/rkaufmann/.ssh/id_rsa": x509: decryption password incorrect

ssh-add                             
Enter passphrase for /home/rkaufmann/.ssh/id_rsa: 
Identity added: /home/rkaufmann/.ssh/id_rsa (/home/rkaufmann/.ssh/id_rsa)


./k3sup install --ip $IP --user root
Public IP: ....
ssh -i /home/rkaufmann/.ssh/id_rsa root@.....
ssh: curl -sLS https://get.k3s.io | INSTALL_K3S_EXEC='server --tls-san xxx.xxx.xx.xx' sh -
....
```

Reply:
So are you saying that you would use `ssh-add` for non-interactive?

Reply:
Yes. What do you think?

Reply:
I think it's workable and a good improvement.

Could you document in the appendix of the README file how to work with an encrypted SSH key?